doc/whats_new/v0.20.rst

```diff
@@ -177,7 +177,10 @@ Miscellaneous
 
 - |Fix| Removed all mentions of `sklearn.externals.joblib`, and deprecated
   joblib methods exposed in `sklearn.utils`, except for
-  :func:`utils.parallel_backend` and :func:`utils.register_parallel_backend`.
+  :func:`utils.parallel_backend` and :func:`utils.register_parallel_backend`,
+  which impact the behavior of `sklearn`. Other functionalities are part of
+  the [joblib](https://joblib.readthedocs.io/) package and should be used
```

Comment:
sorry, @tomMoral I confused you, this is .rst not .md..

Reply:
Is this fix or API?
Also: should we say here that we're planning/considering completely unvendoring in the future?